packages/code/src/main.rs

```diff
@@ -0,0 +1,144 @@
+use std::env;
+use std::fmt::Write;
+use std::io::{self, Read};
+
+use serde_json::{json, Value};
+use syntect::easy::HighlightLines;
+use syntect::parsing::SyntaxSet;
+use syntect::highlighting::{ThemeSet, Color};
+use syntect::html::{styled_line_to_highlighted_html, IncludeBackground};
+
+fn main() {
+    let args: Vec<String> = env::args().skip(1).collect();
+    let action = &args[0];
+    match action.as_str() {
+        "manifest" => manifest(),
+        "transform" => transform(&args[1], &args[2]),
+        other => {
+            eprintln!("Invalid action {other}")
+        }
+    }
+}
+
+fn manifest() {
+    print!("{}", serde_json::to_string(&json!(
+        {
+        "name": "Standard code package",
+        "version": "0.1",
+        "description": "This package provides syntax highlighting in [code] modules",
+        "transforms": [
+            {
+                "from": "code",
+                "to": ["html"],
+                "arguments": [
+                    {"name": "lang", "description": "The language to be highlighted"},
```

Comment:
Hmm maybe have some default language/default to unformatted like \`\`\`-blocks? (Unformatted may be called `text` or something)